src/Nethermind/Nethermind.Runner.Test/EthereumRunnerTests.cs

```diff
@@ -44,6 +48,10 @@ static EthereumRunnerTests()
 
     private static ICollection InitOnce()
     {
+        // we need this to discover ChainSpecEngineParameters
+        new CliqueConfig();
+        new OptimismConfig();
+        new TaikoPlugin();
```

Comment:
Can we do it in some more elegant way?

Reply:
This is the only way I know

Reply:
what are we trying to achieve here?

Reply:
My assumption is that .net is not loading plugins if we don't use them. If you remove this it will not discover `ChainSpecEngineParameters`

Reply:
Can we use `TypeDiscovery` class for that?